gui/velociraptor/src/components/users/user.js

```diff
@@ -0,0 +1,271 @@
+                                          id="verified-email-button"
+                                          value="verified-email"
+                                          type="checkbox">
+                              verified-email
+                            </ToggleButton>
+                          </ToggleButtonGroup>
+                          <ToggleButtonGroup type="checkbox"
+                                             value={[]}>
+                            <ToggleButton onChange={() => {this.setState({read_only: !this.state.read_only});}}
+                                          variant={this.state.read_only ? "default" : "primary"}
+                                          id="read-only-button"
+                                          value="read-only"
+                                          type="checkbox">
+                              read-only
+                            </ToggleButton>
+                          </ToggleButtonGroup>
+                          <ToggleButtonGroup type="checkbox"
+                                             value={[]}>
+                            <ToggleButton onChange={() => {this.setState({locked: !this.state.locked});}}
+                                          variant={this.state.locked ? "default" : "primary"}
+                                          id="locked-button"
+                                          value="locked"
+                                          type="checkbox">
+                              locked
+                            </ToggleButton>
+                          </ToggleButtonGroup>
+                        </InputGroup>
+                      </FormGroup>
+                      <Button onClick={this.handleCreate}
+                              variant="default" type="submit">
+                        {this.state.new_user ? "Add user" : "Apply changes"}
+                      </Button>;
+                    </Form>
+                  </div>
+                </>
+        );
+    }
+};
+
+export const DropdownList = (e) => {
+    const [open, setOpen] = React.useState(false);
+    const onToggle = (isOpen, _, metadata) => {
+        if (metadata.source === "select") {
+            setOpen(true);
+            return;
+        }
+        setOpen(isOpen);
+    };
+
+    const { items, onItemToggle, name } = e;
+    let enabled_items = [];
+
+    let buttons = Object.keys(items).map((item) => {
+        if (items[item]) {
+            enabled_items.push(item);
+        }
+        return <Dropdown.Item key={item}
+                              eventKey={item}
+                              active={items[item]}
+                              onSelect={c=>onItemToggle(c)}>
+                 { item }
+               </Dropdown.Item>;
+    });
+
+    return (
+        <Dropdown show={open} onToggle={onToggle}>
+          <Dropdown.Toggle variant="default" id="dropdown-basic">
```

Comment:
This is generating warnings: `[DOM] Found 2 elements with non-unique id #dropdown-basic: (More info: https://goo.gl/9p2vKq)`